v3/util/time.go

```diff
@@ -42,6 +42,7 @@ var (
 	CABV116Date                 = time.Date(2013, time.July, 29, 0, 0, 0, 0, time.UTC)
 	CABV130Date                 = time.Date(2015, time.April, 16, 0, 0, 0, 0, time.UTC)
 	CABV131Date                 = time.Date(2015, time.September, 28, 0, 0, 0, 0, time.UTC)
+	CABV170Date                 = time.Date(2020, time.January, 31, 0, 0, 0, 0, time.UTC)
```

Comment:
You can see https://github.com/zmap/zlint/pull/449#discussion_r436397811 for some discussion, but I think this might make more sense with a descriptive name about the requirement effective-date rather than the document effective date. WDYT?

Reply:
Okay so I wasn't the _only_ one confused about this (neither the version number nor the ballots were perfectly lining up on this)!

Perhaps a date for the document effective date and aliases for specific requirements?

```go
const (
    // https://cabforum.org/link/to/doc.pdf
    CABV123 = ...
    MustBeTotallyCool = CABV123
    MustBeMostRadical = CABV123
)
```

That is, it signals that these specific requirements were all introduced in this parent document with the breadcrumb to the PDF.

Reply:
Yeah, although it’s possible for one version to have different effective dates for different requirements (like the browser alignment ballot, IIRC), but that SGTM.